modules/seqexec/server/src/main/scala/seqexec/server/altair/AltairEpics.scala

```diff
@@ -80,6 +80,32 @@ class AltairEpics[F[_]: Async](service: CaService, tops: Map[String, String]) {
     s"${AltairTop}cc:lgszoomSfoLoop.VAL", classOf[LgsSfoControl])
   def sfoLoop: F[Option[LgsSfoControl]] = safeAttribute(sfoLoopAttr)
 
+  def aoexpt: F[Option[Double]] = safeAttributeSDouble(status.getDoubleAttribute("aoexpt"))
+
+  def aocounts: F[Option[Double]] = safeAttributeSDouble(status.getDoubleAttribute("aocounts"))
+
+  def aoseeing: F[Option[Double]] = safeAttributeSDouble(status.getDoubleAttribute("aoseeing"))
```

Comment:
Same here, `getDoubleAttribute` will return `null` because the channel was created as Float.